INSTALL

```diff
@@ -1,359 +1,6 @@
-```
-cmake -DCMAKE_TOOLCHAIN_FILE=Platform/BlueGeneQ_GCC \
-      -DCMAKE_INSTALL_PREFIX:PATH=$PWD/install \
-      -Dstatic-libraries=ON -Dwith-warning=OFF \
-      -DCMAKE_EXE_LINKER_FLAGS="-Wl,--allow-multiple-definition" \
-      -Dwith-libraries=$HOME/tcmalloc/install/lib/libtcmalloc.a
-``
-
-
-Compiling for Fujitsu Sparc64
-=============================
-
-On the K Computer:
-  The preinstalled `cmake` version is 2.6, which is too old for NEST. Please install
-  a newer version, e.g. like this:
-
-      wget https://cmake.org/files/v3.4/cmake-3.4.2.tar.gz
-      tar -xzf cmake-3.4.2.tar.gz
-      mv cmake-3.4.2 cmake.src
-      mkdir cmake.build
-      cd cmake.build
-      ../cmake.src/bootstrap --prefix=$PWD/install --parallel=4
-      gmake -j4
-      gmake install
-
-  Also you might need a cross compiled GNU Scientific Library (GSL). For GSL 2.1
-  this is a possible installation scenario:
-
-      wget ftp://ftp.gnu.org/gnu/gsl/gsl-2.1.tar.gz
-      tar -xzf gsl-2.1.tar.gz
-      mkdir gsl-2.1.build gsl-2.1.install
-      cd gsl-2.1.build
-      ../gsl-2.1/configure --prefix=$PWD/../gsl-2.1.install/ \
-                           CC=mpifccpx \
-                           CXX=mpiFCCpx \
-                           CFLAGS="-Nnoline" \
-                           CXXFLAGS="--alternative_tokens -O3 -Kfast,openmp, -Nnoline, -Nquickdbg -NRtrap" \
-                           --host=sparc64-unknown-linux-gnu \
-                           --build=x86_64-unknown-linux-gnu
-      gmake -j4
-      gmake install
-
-  To install NEST, use the following `cmake` line:
-
-      cmake -DCMAKE_TOOLCHAIN_FILE=Platform/Fujitsu-Sparc64 \
-            -DCMAKE_INSTALL_PREFIX:PATH=<nest_install_dir> \
-            -Dwith-gsl=/path/to/gsl-2.1.install/ \
-            -Dwith-optimize="-Kfast -x1" \
-            -Dwith-defines="-DUSE_PMA" \
-            -Dwith-python=OFF \
-            -Dwith-warning=OFF \
-            </path/to/NEST/src>
-      make -j4
-      make install
-
-  The compilation can take quite some time compiling the file `models/modelsmodule.cpp`
-  due to generation of many template classes. To speed up the process, you can
-  comment out all synapse models you do not need.
-  The option `-Kfast` on the K computer enables many different options:
-
-        -O3 -Kdalign,eval,fast_matmul,fp_contract,fp_relaxed,ilfunc,lib,mfunc,ns,omitfp,prefetch_conditional,rdconv -x-
-
-  Be aware that, with the option `-Kfast` an internal compiler error - probably
-  an out of memory situation - can occur. One solution is to disable synapse
-  models that you don't use in `models/modelsmodule.cpp`. From current observations
-  this might be related to the `-x-` option; you can give it a fixed value, e.g
-  `-x1`, and the compilation succeeds (the impact on performance was not analyzed):
-
-        -Dwith-optimize="-Kfast -x1"
-  For NEST 3 or later, C++11 Standard library is required at build time, use `-Xg -std=c++11`.
+or in the doc/installation directory.
```

Comment:
Do any other package managers like to see `INSTALL` around? Otherwise, we could indeed remove it.

I intentionally did not refer to the installed HTML file since I assume that users will want to read the installation documentation before installing ;). So I'd leave this as it is.

Reply:
I think we should rather handle the situation *if* someone complains than have a mostly empty and redundant file lingering around forever.

You're of course totally right with your second point :wink: